core/src/main/java/org/geysermc/geyser/network/CodecProcessor.java

```diff
@@ -271,13 +259,20 @@ static BedrockCodec processCodec(BedrockCodec codec) {
             // Ignored bidirectional packets
             .updateSerializer(ClientCacheStatusPacket.class, IGNORED_SERIALIZER)
             .updateSerializer(SimpleEventPacket.class, IGNORED_SERIALIZER)
-            .updateSerializer(MultiplayerSettingsPacket.class, IGNORED_SERIALIZER);
+            .updateSerializer(MultiplayerSettingsPacket.class, IGNORED_SERIALIZER)
+            // Small limit
+            .updateSerializer(ItemStackRequestPacket.class, ITEM_STACK_REQUEST_SERIALIZER);
+
 
             if (codec.getProtocolVersion() < 685) {
                 // Ignored bidirectional packets
                 codecBuilder.updateSerializer(TickSyncPacket.class, IGNORED_SERIALIZER);
             }
 
+            if (codec.getProtocolVersion() >= 630) { // >= 1.20.50
+                codecBuilder.updateSerializer(SetPlayerInventoryOptionsPacket.class, SET_PLAYER_INVENTORY_OPTIONS_SERIALIZER);
+            }
```

Comment:
I still support 1.20.40+ and I don’t understand why you stopped supporting 1.20.0+ if it’s a proxy server

Reply:
- Bedrock auto-updates. No need to support older versions when anyone using Bedrock from an app store (just about every person not playing cracked, yes?) is on the latest version. (The exception is any Bedrock version that drops older device support, but I am not aware of that happening for the 1.20 versions)
- Each Bedrock version requires more mappings files stored in the jarfile.
- Anything before 1.20.80 doesn't have the new 1.21 blocks. We would have to map those entities and blocks, or ignore them.
- We don't want to test every new feature or packet structure change on older versions to see if they break or crash because they don't support 1.21. In some cases, Java and Bedrock packets or data structures mirror very nicely and we don't want to hack around that for older versions.
- Workarounds can be removed because newer versions work around them - for example, native sneaking and crawling, and better lectern support.